src/Form/Type/Operator/ContainsOperatorType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Form\Type\Operator;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType as FormChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class ContainsOperatorType extends AbstractType
+{
+    public const TYPE_CONTAINS = 1;
+    public const TYPE_NOT_CONTAINS = 2;
+    public const TYPE_EQUAL = 3;
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'choice_translation_domain' => 'SonataAdminBundle',
+            'choices' => [
+                'label_type_contains' => self::TYPE_CONTAINS,
+                'label_type_not_contains' => self::TYPE_NOT_CONTAINS,
+                'label_type_equals' => self::TYPE_EQUAL,
+            ],
+        ]);
+    }
+
+    public function getParent()
+    {
+        return FormChoiceType::class;
+    }
+
+    public function getBlockPrefix()
```

Comment:
Do we need this method?

Reply:
It was in every other form.
It override the implementation of the AbstractType
```
public function getBlockPrefix()
    {
        return StringUtil::fqcnToBlockPrefix(\get_class($this)) ?: ''; // This will be contains_operator_type
    }
```

So I would say yes.

Reply:
It is used as a prefix for the template block in case you want to customize the view of the type, I'm not sure if it makes sense for this type, anyway I guess it is ok in case we need it.